src/components/Reports/WbsTable/WbsTable.jsx

```diff
@@ -1,23 +1,24 @@
 import { Stub } from 'components/common/Stub';
+// eslint-disable-next-line no-unused-vars
```

Comment:
If there is "no-unused-vars", just delete it. You should rarely use "eslint-disable-next-line".

Reply:
fixed